package.json

```diff
@@ -1,29 +1,57 @@
 {
   "name": "videojs-ima",
-  "version": "0.7.0",
+  "version": "0.8.0",
   "license": "Apache-2.0",
-  "main": "./src/videojs.ima.js",
+  "main": "./dist/videojs.ima.js",
   "author": {
     "name": "Google Inc."
   },
   "engines": {
     "node": ">=0.8.0"
   },
   "scripts": {
-    "test": "python -m SimpleHTTPServer"
+    "contBuild": "watch 'npm run rollup:max' src",
+    "devServer": "forever start ./node_modules/http-server/bin/http-server && npm run contBuild",
+    "postdevServer": "forever stop ./node_modules/http-server/bin/http-server",
+    "lint": "eslint \"src/*.js\"",
+    "rollup": "npm-run-all rollup:*",
+    "rollup:max": "rollup -c configs/rollup.config.js",
+    "rollup:min": "rollup -c configs/rollup.config.min.js",
+    "preversion": "npm run lint",
+    "version": "node scripts/version.js",
+    "postversion": "node scripts/postversion.js"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/googleads/videojs-ima"
   },
+  "files": [
+    "CHANGELOG.md",
+    "LICENSE",
+    "README.md",
+    "dist/"
+  ],
   "dependencies": {
-    "videojs-contrib-ads": "~5.0.3"
+    "video.js": ">=5.19.2",
+    "videojs-contrib-ads": "~5.1.0"
   },
   "devDependencies": {
-    "grunt": "^0.4.5",
-    "videojs-doc-generator": "0.0.1"
-  },
-  "peerDependencies": {
-    "video.js": ">=5.3.0"
```

Comment:
This is a regression of #417 is there a reason that video.js can't be a peer dependency anymore? Seems weird that a plugin for a thing should also include the thing. Thoughts?